### PR TITLE
Remove kube-scheduler-policy contents

### LINKED ISSUE #31005: Scheduler policy API is removed in 1.23
**This is a Bug Report**

**Problem:**

The scheduler policy API was completely removed in 1.23.

**Proposed Solution:**

- Revise concepts/configuration/manage-resources-containers.md to remove references to the scheduler policy API;
- Revise or even remove reference/scheduling/policies.md which is meaningless;
- Remove the kube-scheduler-policy-config.v1.md file from config API.

**Page to Update:**

See above.

Related: #26450

### PR DESCRIPTION
**Pull request by tengqm**
closes #31005

### CODE REVIEW

File: content/en/docs/concepts/configuration/manage-resources-containers.md [hunk 1]
```diff
@@ -761,5 +761,5 @@ You can see that the Container was terminated because of `reason:OOM Killed`, wh
 * Read the [Container](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#container-v1-core) API reference
 * Read the [ResourceRequirements](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#resourcerequirements-v1-core) API reference
 * Read about [project quotas](https://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html) in XFS
-* Read more about the [kube-scheduler Policy reference (v1)](/docs/reference/config-api/kube-scheduler-policy-config.v1/)
+* Read more about the [kube-scheduler configuraiton reference (v1beta3)](/docs/reference/config-api/kube-scheduler-config.v1beta3/)
```
Comment (sftim) [inline content/en/docs/concepts/configuration/manage-resources-containers.md:764]: nit:
```suggestion
* Read more about the [kube-scheduler configuration reference (v1beta3)](/docs/reference/config-api/kube-scheduler-config.v1beta3/)
```

**Review by sftim — COMMENTED** [review]
/approve

(one spelling nit)

File: content/en/docs/concepts/configuration/manage-resources-containers.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/configuration/manage-resources-containers.md:764]: whoops ...

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-31006--kubernetes-io-main-staging.netlify.app/docs/concepts/configuration/manage-resources-containers/#what-s-next
https://deploy-preview-31006--kubernetes-io-main-staging.netlify.app/docs/reference/scheduling/policies/#what-s-next
https://deploy-preview-31006--kubernetes-io-main-staging.netlify.app/docs/reference/#config-apis

**Pull request merged**